criu/seize.c

```diff
@@ -24,6 +24,46 @@
 #include "xmalloc.h"
 #include "util.h"
 
+char *task_comm_info(pid_t pid, char *comm, size_t size)
+{
+	int ret = 0;
+
+	if (!pr_quelled(LOG_INFO)) {
+		int saved_errno = errno;
+		char path[32];
+		int fd;
+
+		snprintf(path, sizeof(path), "/proc/%d/comm", pid);
+		fd = open(path, O_RDONLY);
+		if (fd >= 0) {
+			ssize_t n = read(fd, comm, size);
+			if (n > 0)
+				comm[n - 1] = '\0';
+			else
+				ret = -1;
+			close(fd);
+		} else {
+			ret = -1;
```

Comment:
it is better to print an error here

Reply:
Added warnings in scope of https://github.com/checkpoint-restore/criu/pull/2095